docs/changelog.md

```diff
@@ -4,6 +4,9 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## v2.0.2 - TBA 
+* Added `BOOTSTRAP_SERVERS` option to `CREATE STREAM` which you can check [here.](reference-guide/streams/overview.md)
```

Comment:
Add PR link which introduced that change. Also rebase.

Reply:
done!